### ISSUE TITLE
Synchronization errors

### ISSUE THREAD

**Issue by TBotV63** [meguy26/RemoteConsole#10]
```Java
public synchronized Game getGame(){
	return this.game;
}
```

This is flawed as it synchronizes on the plugin's main class. Synchronization should occur on the `Game` class itsself and is something the threads need to take care of theirselves.

**Comment by meguy26** [comment]
Yeah. I use that to exec commands and pass plugins. I could pass a list of plugin names to the external client class, as well as start a sync task from the thread to exec a command. In theory, creating and running a sync task from another thread will still cause it to run on the main thread.
